server/database/schema.js

```diff
@@ -1,6 +1,7 @@
 import { MongoClient, ServerApiVersion } from 'mongodb';
 import 'dotenv/config';
 import { nanoid } from 'nanoid';
+import { TOPICS } from './dummyDataERD.js';
```

Comment:
Is `TOPICS` the right import?

```suggestion
import { insertERDiagramsData } from './dummyDataERD.js';
```

Reply:
omg sorry I'm in a rush. I fixed it now